Remotion/Web/Development.WebTesting.IntegrationTests/MouseTest.cs

```diff
@@ -102,6 +102,8 @@ public void Test_Focus ()
 
       var focusElement = home.Scope.FindId(c_focusDivID);
 
+      Thread.Sleep(250);
```

Comment:
we need to figure out how to solve this without adding a thread-sleep in the test. normally, this indicates that there is an underlying issue that results in a systematic delay. if that's the case, everyone would be affected.

Reply:
@a-ctor got ideas how to best analyze/solve this?

Reply:
We can use `RetryUnilTimeout.Run(...)` instead of waiting. Only downside is that that same assertion error is logged multiple times. So if the test fails the error is printed about 15 times. We can either ignore this or use NUnit's `IsolatedConext` in the delegate to prevent this.

Reply:
@a-ctor the problem is, if the test is flaky, it's reasonably to assume that this will also affect users of the library. we should first find out why the test is flaky and if we can do something in the webtest library to ensure users remain unaffected. Only for cases where our own setup is causing the instability should we use Retry/delay in the test code.